productivity/mock.py

```diff
@@ -63,6 +64,10 @@ async def _request(self, method, *args, **kwargs):
             for i, d in enumerate(data):
                 self._coils[address + i] = d
             return WriteMultipleCoilsResponse(address, len(data))
+        elif method == 'write_register':
+            address, data = args
+            self._coils[address] = data
```

Comment:
I think you mean '_registers' here

Reply:
No, I mean `write_register` since `write_registers` already exists below it...?

Reply:
```suggestion
            self. _registers[address] = data
```

Reply:
d'oh